templates/class/default.tmpl.js

```diff
@@ -0,0 +1,106 @@
+${header}
+
+/**
+ * This is a qooxdoo class
+ *
+ * //@asset(${namespace}/*)
+ * //@require(${namespace}/*)
+ */
+qx.Class.define("${classname}",
+{
+  
+  extend : ${extend},
+  include : [],
+
+  /*
+  *****************************************************************************
+     CONSTRUCTOR
+  *****************************************************************************
+  */
+
+  /**
+   * Create a new instance
+   */
+  construct : function() {
+  	
+  },
+
+/*
+  *****************************************************************************
+     PROPERTIES
+  *****************************************************************************
+  */
+
+  properties :
+  {
+    /** The foo property of the object */
+    foo :
+    {
+      apply : "_applyFoo",
+      nullable : true,
+      check : "String",
+      event : "changeFoo"
+    }
+  },
+  
+
+  /*
+  *****************************************************************************
+     EVENTS
+  *****************************************************************************
+  */
+
+  events :
+  {
+    /** Fired when something happens */
+    "changeSituation" : "qx.event.type.Data"
+  },  
+
+  /*
+  *****************************************************************************
+     STATICS
+  *****************************************************************************
+  */
+
+  statics :
+  {
+    /** The foo static property of the class object */
+    FOO : "bar"
+  },  
+
+  /*
+  *****************************************************************************
+     MEMBERS
+  *****************************************************************************
+  */
```

Comment:
Do you guys actually retain these comment blocks for MEMBER, EVENTS, etc? The first thing I do, if copying something from a framework class, is get rid of that cruft. It's marked as `members :` immediately below, so really doesn't add any benefit at all, as far as I can see...

Reply:
👍

Reply:
Well, I think the 1&1 guys added the comments for better readability of long classes (you can jump much faster between code blocks with these visual markers), and it still might some value for beginners. But I certainly don't insist on keeping them.

Reply:
I just asked the question in the core-team chat, but maybe that would also be a question for the community... Is that a show stopper for you or could we go with the verbose templates for the moment? We can still change them later and in any case I don't think you will use the standard templates anyways, but create your own ones if you use them at all.